asciidoctorj-core/src/main/java/org/asciidoctor/ast/SectionImpl.java

```diff
@@ -1,39 +1,37 @@
 package org.asciidoctor.ast;
 
 import org.jruby.Ruby;
+import org.jruby.runtime.builtin.IRubyObject;
 
 public class SectionImpl extends AbstractBlockImpl implements Section {
```

Comment:
We should add "get" methods in Section and SectionImpl and deprecate the methods that don't start with "get".